tinker/libs/bunq_lib.py

```diff
@@ -1,13 +1,14 @@
 import json
 from os.path import isfile
+from os import remove
 import socket
 
 import requests
 from bunq.sdk.client import Pagination
 from bunq.sdk.context import ApiContext
 from bunq.sdk.context import ApiEnvironmentType
 from bunq.sdk.context import BunqContext
-from bunq.sdk.exception import BunqException
+from bunq.sdk.exception import BunqException, ForbiddenException
```

Comment:
Separate these imports please.

Imports should usually be on separate lines, e.g.:

> Yes: import os
>      import sys
> 
> No:  import sys, os
https://www.python.org/dev/peps/pep-0008/#imports